.github/workflows/release.yaml

```diff
@@ -0,0 +1,100 @@
+on:
+  release:
+    types: [published]
+
+name: Publish Bundle
+
+jobs:
+  build:
+    name: Publish Bundle
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Get Release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.ACTIONS_TOKEN }}
+
+      - name: Generate Bundle
+        run: |
+          git submodule init
+          git submodule update
+          docker build -t common-templates-ci ./builder/
+          docker run \
+            -v "$(pwd)":/common-templates \
+            common-templates-ci \
+            /bin/bash -c "cd /common-templates && export REVISION=1 && make release"
+
+      - name: Upload Release Asset
+        id: upload-bundle-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.ACTIONS_TOKEN }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
+          asset_path: ./dist/common-templates.yaml
+          asset_name: common-templates.yaml
+          asset_content_type: text/plain
+
+      - name: Update SSP Operator
+        run: |
+          # Install required packages
+          curl -L https://golang.org/dl/go1.16.linux-amd64.tar.gz -o go.tar.gz
```

Comment:
Maybe you can use `actions/setup-go` to install golang. https://github.com/marketplace/actions/setup-go-environment

Reply:
Yes, that's a great idea! I will update the code